sdk/tests/conformance/extensions/ext-disjoint-timer-query.html

```diff
@@ -78,6 +83,9 @@
         verifyQueryResultsNotAvailable();
 
         window.requestAnimationFrame(checkQueryResults);
+
+        if (window.parent.webglTestHarness)
+            window.parent.webglTestHarness.setTimeoutDelay(20000);
```

Comment:
This is very flaky. In case we decide to change the global timeout delay, we most likely won't remember this place.

Is there a way to have a general mechanism to mark the test as "slow" and automatically double the timeout (and set it back afterwards)?